### ISSUE TITLE
SPIKE - Impact of removing hate crime question from form

### ISSUE THREAD

**Issue by blaughman** [usdoj-crt/crt-portal-management#1090]
Devs will assess and document any impact on the system (tests, etc.) resulting from removing the hate crime question from the complaint form.

**Comment by louh** [comment]
The draft PR linked above is a "minimum viable" removal of the hate crime step, with E2E tests updated to pass on CI. It may be worth doing a manual bug-bashing review, but so far the impact on the overall system would appear to be quite low.

When we proceed with the full removal, these are follow up actions to take:
- Remove code and content that have become unused (e.g. in form fields, templates, translations, and unit tests). (We discussed whether or not we want to keep them around "just in case" it comes back, but we can always pull it back out from git history. Leaving unused code and content around becomes a maintenance burden.)
  - Note: I'm assuming that we are not deleting any data already collected with this question, so hate crimes remains in the model, and can still be displayed and filtered on in the intake specialist views.
  - We are also preserving the modal code and content from the National Human Trafficking Hotline link, which will be transferred to the new hate crimes page.
- Locally run pa11y tests were not updated here because they are not running in CI, but should be updated as well.

**Comment by blaughman** [comment]
@louh Excellent job!

**Comment by randyabramson** [comment]
Thanks @louh  - yes, one of the concerns was that removing this question could somehow impact past reports labeled as hate crime, etc., but it seems like we are OK here, correct here, right?

**Comment by louh** [comment]
@randyabramson That's right. We can safely remove the hate crime question from the public-facing complaint form, and it won't affect past reports or internal workflows.
